src/app/orbit-counts/orbit-counts.component.ts

```diff
@@ -7,12 +7,13 @@ import { Satellite } from '../satellite';
   styleUrls: ['./orbit-counts.component.css']
 })
 export class OrbitCountsComponent implements OnInit {
-
+	types = ['Communication', 'Probe', 'Space Station', 'Telescope', 'Space Debris', 'Positioning'];
```

Comment:
This is great, but with this being a typescript file, and a typed language, I would suggest adding the type here:

`types : string[] = ['Communication','Probe','Space Station','Telescope','Space Debris','Positioning'];`